File: WinForms/VB/PdfViewerDemo/MainForm.vb
Imports System.ComponentModel
Imports System.Windows.Forms
Imports DevExpress.DXperience.Demos
Imports DevExpress.LookAndFeel
Imports DevExpress.XtraEditors

Namespace DevExpress.XtraPdfViewer.Demos

    Public Partial Class MainForm
        Inherits RibbonMainForm

        Public Overrides ReadOnly Property UseEmptyRootPadding As Boolean
            Get
                Return True
            End Get
        End Property

        Protected Overrides ReadOnly Property CreateFeedbackPanel As Boolean
            Get
                Return False
            End Get
        End Property

        Protected Overrides ReadOnly Property ShowPanelDescription As Boolean
            Get
                Return False
            End Get
        End Property

        Protected Overrides ReadOnly Property SupportAdvancedTitlePainting As Boolean
            Get
                Return True
            End Get
        End Property

        Protected Overrides ReadOnly Property ProductName As String
            Get
                Return "XtraPdfViewer"
            End Get
        End Property

        <DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)>
        Public Overrides Property NavigationControl As Control
            Get
                Return Nothing
            End Get

            Set(ByVal value As Control)
            End Set
        End Property

        Public Sub New()
            InitializeComponent()
        End Sub

        Protected Overrides Sub ShowModule(ByVal name As String, ByVal group As GroupControl, ByVal lookAndFeel As DefaultLookAndFeel)
            DemosInfo.DoShowModule(name, group, RibbonMenuManager)
        End Sub

        Protected Overrides Sub ShowAbout()
            PdfViewer.About()
        End Sub
    End Class
End Namespace

File: WinForms/CS/PdfViewerDemo/MainForm.cs
using System.ComponentModel;
using System.Windows.Forms;
using DevExpress.DXperience.Demos;
using DevExpress.LookAndFeel;
using DevExpress.XtraEditors;

namespace DevExpress.XtraPdfViewer.Demos {
    public partial class MainForm : RibbonMainForm {
        public override bool UseEmptyRootPadding { get { return true; } }
        protected override bool CreateFeedbackPanel { get { return false; } }
        protected override bool ShowPanelDescription { get { return false; } }
        protected override bool SupportAdvancedTitlePainting { get { return true; } }
        protected override string ProductName { get { return "XtraPdfViewer"; } }

        [DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden)]
        public override Control NavigationControl {
            get { return null; }
            set { }
        }

        public MainForm() {
            InitializeComponent();
        }
        protected override void ShowModule(string name, GroupControl group, DefaultLookAndFeel lookAndFeel) {
            DemosInfo.DoShowModule(name, group, RibbonMenuManager);
        }
        protected override void ShowAbout() {
            DevExpress.XtraPdfViewer.PdfViewer.About();
        }
    }
}
